src/java/org/apache/cassandra/index/sai/memory/TrieMemoryIndex.java

```diff
@@ -128,6 +130,12 @@ public synchronized Map<PrimaryKey, Integer> getDocLengths()
         return docLengths;
     }
 
+    @Override
+    public int size()
+    {
+        return docCount.get();
```

Comment:
Can we do something to the naming here to make it clear this is number of items and not a size in bytes?  I can see that being confused in the future.

Reply:
renamed to `indexedRows`